ICAFIX/ReApplyFixMultiRunPipeline.sh

```diff
@@ -415,22 +452,16 @@ main()
 	if [ -z "${9}" ]; then
 		MotionRegression="${G_DEFAULT_MOTION_REGRESSION}"
```

Comment:
If using the default setting, `MotionRegression` will be set to `FALSE`, and never gets converted to a boolean (as is neccessary for fix_3_clean).  In the previous code, the conversion to boolean happened after MotionRegression was already defined, regardless of whether we were setting to `G_DEFAULT_MOTION_REGRESSION`

Reply:
These should be resolved by f137e43d94bbafbdb7aced00dae753bdbbd2aca8, including in single-run reapply.  The hcp_fix* versions should work as-is, because their "interpret_as_bool" actually outputs strings rather than numbers, because that is what they expected (and I don't want to reengineer their existing boolean handling for this minor change).